packages/framer-motion/src/animation/sequence/__tests__/index.test.ts

```diff
@@ -493,9 +494,9 @@ describe("createAnimationsFromSequence", () => {
         const { duration, ease, times } = animations.get(a)!.transition.x
 
         expect(duration).toEqual(2)
-        expect(ease![0]).toEqual("linear")
-        expect(ease![1]).toEqual("linear")
-        expect(typeof ease![2]).toEqual("function")
+        expect((ease as Easing[])[0]).toEqual("linear")
```

Comment:
@mattgperry 
There is degradation in type-checking.
It’s safer to use non-null assertion than `as`.
https://www.reddit.com/r/typescript/comments/11rqtn3/is_as_any_better_than_to_assert_nonnullishness/